### ISSUE TITLE
-v does not appear to be respected

### ISSUE THREAD

**Issue by JohnEmhoff** [Microsoft/pyright#9]
This small script creates a virtual environment, installs `requests`, and then shows that pyright complains that `'requests' could not be resolved`:

```bash
#!/bin/bash

virtualenv env
source env/bin/activate
pip3 install requests

cat > test.py <<EOF
import requests
EOF

pyright -v env test.py
```

Output:

```
Finding source files
Found 1 source files
Analyzed 1 file in 1.414sec
/home/john/test/test.py
  Import 'requests' could not be resolved (1, 8)
1 error, 0 warnings
```

I could be using `-v` incorrectly (the documentation is a little unclear), but this usage seems like the most natural by far. Giving garbage paths to `-v` also does not trigger an error, leading me to believe it just isn't used?

**Comment by erictraut** [comment]
The `-v` option is meant to point to a directory on your system that contains your virtual environments. It's used in conjunction with the pyrightconfig.json file, which refers to individual venvs by name. The idea here is that everyone on a team can share the same pyrightconfig.json (which is then checked into the repo), but each person can place their venvs in a different location on their local machine.

I'll clarify this in the documentation.
